cmd/ddev/cmd/import-files.go

```diff
@@ -30,8 +33,8 @@ provided if it is not located at the top-level of the archive. If the
 destination directory exists, it will be replaced with the assets being
 imported.
 
-The destination directory can be configured in your project's config.yaml
-under the upload_dir key. If no custom upload directory is defined, the app
+The destination directories can be configured in your project's config.yaml
+under the upload_dirs key. If no custom upload directory is defined, the app
```

Comment:
If I'm not mistaken, this needs examples of doing import-files with multiple upload_dirs.

```
ddev import-files --src=.tarballs/files.tar.xz --target=../private
ddev import-files --src=.tarballs/files.tar.gz --target=sites/default/files
```

Reply:
Added to the examples above